tools/sideloading/README.md

```diff
@@ -0,0 +1,229 @@
+    * Option 2 (advanced) : Deploy using AWSCLI or API.
+        1. Include the "networkRoutingRules" element in the override manifest file as well.
+            ``` json
+            {
+                "nodeGraphOverrides":{
+
+                    ... snip ...
+
+                    "networkRoutingRules":[
+                        {
+                            "node": "code_node",
+                            "containerPort": 8123,
+                            "hostPort": 8123
+                        }
+                    ]
+                }
+            }
+            ```
+            **Note:** Please make sure you use right code node name. In this example, "code_node" is specified as the code node name, but it can be different in your manifest file.
+
+        1. Deploy the application programmatically using boto3's create_application_instance() API or AWSCLI's "aws panorama create-application-instance" command. Specify contents of graph.json as the manifest payload, and contents of override.json as the override manifest payload. (See also : [Automate application deployment](https://docs.aws.amazon.com/panorama/latest/dev/api-deploy.html)).
+
+1. Confirm the completion of deployment.
+    * **Note** : After the deployment completed, you will see `Error` or `Not available` status. You can safely ignore these errors. It just means the application hasn't started regular computer vision process yet.
+
+1. Check "console_output" log stream on CloudWatch Logs, and confirm "SideloadingAgent started" message.
+
+1. Identify the IP address of your appliance device. You can use the Management Console UI or "aws panorama describe-device" command.
+
+1. Under the top directory of your application (where assets / graphs / packages directories exist), hit "`sideloading_cli.py sync`" command to transfer updated files. Following is an example:
+
+    ```bash
+    $ python3 aws-panorama-samples/tools/sideloading/sideloading_cli.py sync --addr 10.0.0.245 --port 8123 --src-top packages/123456789012-mycode-1.0/src
+    ```
+
+    **Note:** This is an example. Please replace script path, IP address of the device, port number, and path to the "src" directory according to your environment.
+
+    **Note:** By default, sideloading_cli.py tries to find PEM files from current directory. Alternatively you can specify the path with `--cert_key_dir` option.
+
+1. Under the top directory of your application (where assets / graphs / packages directory exist), hit "`sideloading_cli.py run-app`" command to run application with transfered files. Following is an example:
+
+    ```bash
+    $ python3 aws-panorama-samples/tools/sideloading/sideloading_cli.py run-app --addr 10.0.0.245 --port 8123
+    ```
+
+1. Reboot the device, and repeat "sync" and "run-app".
+
+
+## How to switch to Production mode
+
+When you completed the development and are ready to deploy to production system, please take following steps.
```

Comment:
add step to remove the certs, specify exact steps

Reply:
Adding this line:

> Remove `sideloading_*.pem` files from the code package src directory.

Reply:
Also modifying security best practice section as below:

> When you deploy your application to Production environment, follow the steps in [How to switch to Production mode](#how-to-switch-to-production-mode) in this docment.